### PR TITLE
added alumni to member button

### PR DESCRIPTION
**Pull request by Arteox**
Added a new button for updating the status of an alumni back to a member

### CODE REVIEW

**Review by teckwan — CHANGES_REQUESTED** [review]
if you can, please change the settings of your IDE, change tabs into spaces, and make each tab 2 spaces.

**Comment by Arteox** [comment]
> if you can, please change the settings of your IDE, change tabs into spaces, and make each tab 2 spaces.

Ok will do !

File: views/partials/info-box/profile/buttons.hbs [hunk 1]
```diff
@@ -22,6 +22,8 @@
   {{/if}}
   {{#if (eq entity "members")}}
     {{> modals/alumni}}
+  {{else if (eq entity "alumni")}}
+      {{> modals/member}}
```
Comment (teckwan) [inline views/partials/info-box/profile/buttons.hbs]: not a 2 space tab i think....

File: views/partials/modals/member.hbs [hunk 2]
```diff
@@ -0,0 +1,50 @@
+<button
+  type="button"
+  class="btn btn-info text-light mx-1"
+  data-toggle="modal"
+  data-target="#backToMemberModal"
+>
+    <i class="fas fa-angle-double-right mr-1"></i>
+    <i class="fas fa-walking"></i>
```
Comment (teckwan) [inline views/partials/modals/member.hbs]: here as well

File: views/partials/modals/member.hbs [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+          class="close"
+          data-dismiss="modal"
+          aria-label="close"
+        >
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+        <p class="text-center">
+        Êtes-vous sûr de vouloir passer l'ancien en membre ?
```
Comment (teckwan) [inline views/partials/modals/member.hbs]: one tab here if possible

**Review by teckwan — CHANGES_REQUESTED** [review]
Sorry I'm a little annoying, just trying to make everything consistent

**Pull request merged**